lib/gui/etcher.ts

```diff
@@ -206,32 +205,6 @@ async function createMainWindow() {
 		)}`,
 	);
 
-	const page = mainWindow.webContents;
-
-	page.once('did-frame-finish-load', async () => {
-		console.log('packageUpdatable', packageUpdatable);
-		autoUpdater.on('error', (err) => {
-			logMainProcessException(err);
-		});
-		if (packageUpdatable) {
-			try {
-				const configUrl = await settings.get('configUrl');
-				const onlineConfig = await getConfig(configUrl);
-				const autoUpdaterConfig: AutoUpdaterConfig = onlineConfig?.autoUpdates
-					?.autoUpdaterConfig ?? {
-					autoDownload: false,
-				};
-				for (const [key, value] of Object.entries(autoUpdaterConfig)) {
-					autoUpdater[key as keyof AutoUpdaterConfig] = value;
-				}
-				const checkForUpdatesTimer =
-					onlineConfig?.autoUpdates?.checkForUpdatesTimer ?? 300000;
-				checkForUpdates(checkForUpdatesTimer);
```

Comment:
Looks like this also removes the call to `checkForUpdates` - was that intentional?

Reply:
aouch! No not intentional. Good catch!

Reply:
fixed in https://github.com/balena-io/etcher/pull/4006